server/sonar-web/src/main/js/apps/issues/components/BulkChangeModal.js

```diff
@@ -0,0 +1,522 @@
+      remove_tags: this.state.removeTags && this.state.removeTags.join(),
+      do_transition: this.state.transition,
+      comment: this.state.comment,
+      sendNotifications: this.state.notifications
+    });
+    const issueKeys = this.state.issues.map(issue => issue.key);
+
+    this.setState({ submitting: true });
+    bulkChangeIssues(issueKeys, query).then(
+      () => {
+        this.setState({ submitting: false });
+        this.props.onDone();
+      },
+      (error: Error) => {
+        this.setState({ submitting: false });
+        this.props.onRequestFail(error);
+      }
+    );
+  };
+
+  getAvailableTransitions(issues: Array<Issue>): Array<{ transition: string, count: number }> {
+    const transitions = {};
+    issues.forEach(issue => {
+      if (issue.transitions) {
+        issue.transitions.forEach(t => {
+          if (transitions[t] != null) {
+            transitions[t]++;
+          } else {
+            transitions[t] = 1;
+          }
+        });
+      }
+    });
+    return sortBy(Object.keys(transitions)).map(transition => ({
+      transition,
+      count: transitions[transition]
+    }));
+  }
+
+  renderCancelButton = () => (
+    <a id="bulk-change-cancel" href="#" onClick={this.handleCloseClick}>
+      {translate('cancel')}
+    </a>
+  );
+
+  renderLoading = () => (
+    <div>
+      <div className="modal-head">
+        <h2>{translate('bulk_change')}</h2>
+      </div>
+      <div className="modal-body">
+        <div className="text-center">
+          <i className="spinner spinner-margin" />
+        </div>
+      </div>
+      <div className="modal-foot">
+        {this.renderCancelButton()}
+      </div>
+    </div>
+  );
+
+  renderCheckbox = (field: string) => (
+    <Checkbox
+      className={css({ paddingTop: 6, paddingRight: 8 })}
+      checked={this.state[field] != null}
+      onCheck={this.handleFieldCheck(field)}
+    />
+  );
+
+  renderAffected = (affected: number) => (
+    <div className="pull-right note">
+      ({translateWithParameters('issue_bulk_change.x_issues', affected)})
+    </div>
+  );
+
+  renderField = (field: string, label: string, affected: ?number, input: Object) => (
+    <div className="modal-field">
+      <label htmlFor={field}>{translate(label)}</label>
+      {this.renderCheckbox(field)}
+      {input}
+      {affected != null && this.renderAffected(affected)}
+    </div>
+  );
+
+  renderAssigneeOption = (option: { avatar?: string, email?: string, label: string }) => (
+    <span>
+      {(option.avatar != null || option.email != null) &&
+        <Avatar
+          className="little-spacer-right"
+          email={option.email}
+          hash={option.avatar}
+          size={16}
+        />}
+      {option.label}
+    </span>
+  );
+
+  renderAssigneeField = () => {
+    const affected: number = this.state.issues.filter(hasAction('assign')).length;
+
+    if (affected === 0) {
+      return null;
+    }
+
+    const input = (
+      <SearchSelect
```

Comment:
I'm not sure we can, I was just thinking that we might be able to use the `SearchSelect` in the `SetAssigneePopup` since their final result is exactly the same.